src/main/java/rx/subscriptions/CompositeSubscription.java

```diff
@@ -116,6 +118,11 @@ public void clear() {
         }
     }
 
+    /**
+     * Unsubscribes itself and all inner subscriptions.
+     * <p>Notice that {@link CompositeSubscription} won't be able
+     * to manage new subscriptions after call of this method.
```

Comment:
I feel this is unclear. You can still add `Subscription` to a `CompositeSubscription` after call of this method. How about `After call of this method, new {@code Subscription}s added to {@link CompositeSubscription} will be unsubscribed immediately`?